packages/sdk/typescript/human-protocol-sdk/scripts/run-test.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
+set -eux
```

Comment:
all this scripts does can be done from package.json, check fortune for an example

Reply:
Is there any benefit of having those complex commands directly in package.json? I prefer using bash script which is clean, and better to manage.